### ISSUE TITLE
authStore.isValid always false

### ISSUE THREAD

**Issue by iskaa02** [pocketbase/js-sdk#42]
isValid method always returns false.

I'm using React native since it doesn't have compatibility with all the web APIs this was an issue, it's because of the [`atop()` function](https://developer.mozilla.org/en-US/docs/Web/API/atob) used in the [`isTokenExpired`](https://github.com/pocketbase/js-sdk/blob/df56023da5383ecdd86eafa7be875771e759f0e8/src/stores/utils/jwt.ts#L11-L25 ) function.

I slightly modified the implementation to use [`jwt_decode`](https://github.com/auth0/jwt-decode) library and it's working great for me.  
```typescript
function isTokenExpired(token: string, expirationThreshold = 0): boolean {
  let payload: { [x: string]: any } = jwt_decode(token);

  if (
    Object.keys(payload).length > 0 &&
    (!payload.exp || payload.exp - expirationThreshold > Date.now() / 1000)
  ) {
    return false;
  }

  return true;
}
```

**Comment by ganigeorgiev** [comment]
What version of nodejs are you using?

That's strange because we have a polyfill for the `atob` in https://github.com/pocketbase/js-sdk/blob/master/src/stores/utils/jwt.ts#L5). Maybe the `Buffer` definition is not available? I'll test it later today after work.

**Comment by ganigeorgiev** [comment]
@iskaa02 I'm not able to reproduce the issue (at least when I test with https://snack.expo.dev/)

Are you testing on a real device or emulator?

**Comment by iskaa02** [comment]
Testing on real Android 9.0 device.

**Comment by ganigeorgiev** [comment]
@iskaa02 This should be now fixed in the latest [v0.7.3 release](https://github.com/pocketbase/js-sdk/releases/tag/v0.7.3).

If you are still experiencing issues after the update, please let me know and I'll investigate it further.
